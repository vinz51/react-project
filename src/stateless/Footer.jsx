import React from 'react'
import * as Classes from 'Styles/components.footer'

const Footer = () => (
    <div className={Classes.footer}>
        <p className={Classes['footer-text']}>Website created by Vincent BENOIT - 2018</p>
    </div>
);

export default Footer
