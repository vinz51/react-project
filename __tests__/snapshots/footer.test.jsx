import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../../src/stateless/Footer'

it('snapshot footer', () => {
    const footer = renderer.create(<Footer />).toJSON()
    expect(footer).toMatchSnapshot()
})
