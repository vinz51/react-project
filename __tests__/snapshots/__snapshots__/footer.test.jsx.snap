// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot footer 1`] = `
<div
  className={undefined}
>
  <p
    className={undefined}
  >
    Website created by Vincent BENOIT - 2018
  </p>
</div>
`;
